### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by derekendres**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ Developed against DotNet 5.0.
 
 ## About this sample
 
+
+| :loudspeaker: **Notice**: The PI Adapter for Structured Data Files can send CSVs (and other files types) to EDS, OCS, and PI.  https://osisoft.github.io/PI-Adapter-Structured-Data-Files-Docs/content/index.html.  It has many more features than this sample.|
```
Comment (dcbroad3) [inline README.md]: If I may nitpick, you've got an extra space between each sentence.

Comment (dcbroad3) [inline README.md]: Consider making this an inline link in the previous sentence? Like `The [PI Adapter for Structured Data Files](https://osisoft.github.io/PI-Adapter-Structured-Data-Files-Docs/content/index.html) can send..`

Comment (derekendres) [reply README.md]: sounds good.  i also changed my 2nd sentance.  I want to encourage people to use the adapter, but in talking with the Product team they think it is good to keep the sample out there too

**Review by dcbroad3 — APPROVED** [review]

**Comment by derekendres** [comment]
@kelseyduffy @cbardini I got confirmation from Jiyeon and Felts and Laurent this is the general path to go down

**Review by kelseyduffy — APPROVED** [review]

**Pull request merged**